compiler-rt/test/ubsan/TestCases/Integer/bit-int.c

```diff
@@ -1,3 +1,4 @@
+// REQUIRES: x86_64-target-arch
```

Comment:
What is about 
https://lab.llvm.org/buildbot/#/builders/107/builds/1916

Reply:
I will disabled for windows as well, please investigate after we fix bots?

Reply:
It seems the test does pass when there are 128-bit runtime routines available on Windows, which means the XFAIL breaks: https://logs.chromium.org/logs/chromium/buildbucket/cr-buildbucket/8739197731419270081/+/u/package_clang/stdout

I'll add `REQUIRES: !windows` for now instead.